src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -85,6 +85,14 @@ public override string StringValue
                 }
             }
 
+            internal override Rope RopeValue
```

Comment:
```suggestion
            internal override Rope RopeValue => null;
```

Reply:
I would do that if all the surrounding ones used that style.

---
In reply to: [313170693](https://github.com/dotnet/roslyn/pull/37915#discussion_r313170693) [](ancestors = 313170693)